custom-configurations/src/main/java/com/sequenceiq/cloudbreak/validation/AllRoleTypes.java

```diff
@@ -0,0 +1,65 @@
+package com.sequenceiq.cloudbreak.validation;
+
+public enum AllRoleTypes {
+    CRUISE_CONTROL_SERVER,
```

Comment:
dont we have this information in the cM client? In that case we dont need this code

Reply:
I added this and the other enum for validation so that the frontend can limit the user input for service and role type. Would the CM API be able to fetch all the available services and roles in all the cluster templates?